windows/src/main/csharp/ch/cyberduck/ui/ViewModels/TransfersViewModel.cs

```diff
@@ -0,0 +1,535 @@
+        {
+            if (!item.Running)
+            {
+                TransferOptions options = new TransferOptions()
+                    .resume(true)
+                    .reload(false);
+                controller.StartTransfer(item.Transfer.Model, options);
+            }
+        }
+    }
+
+    private void OnSelectedBandwidthChanged(BandwidthViewModel value)
+    {
+        foreach (var transfer in SelectedTransfers)
+        {
+            transfer.Transfer.Model.setBandwidth(value.Bandwidth);
+            if (transfer.Running && controller.Lookup(transfer.Transfer.Model) is { Action: { } action })
+            {
+                action.getMeter().reset();
+            }
+
+            transfer.Transfer.Refresh();
+        }
+    }
+
+    private void OnSelectedTransferPropertyChanged(TransferViewModel _)
+    {
+        OpenCommand.NotifyCanExecuteChanged();
+        ReloadCommand.NotifyCanExecuteChanged();
+        ResumeCommand.NotifyCanExecuteChanged();
+        ShowCommand.NotifyCanExecuteChanged();
+        StopCommand.NotifyCanExecuteChanged();
+        TrashCommand.NotifyCanExecuteChanged();
+    }
+
+    private void OnSelectionCountChanged(int _)
+    {
+        OpenCommand.NotifyCanExecuteChanged();
+        OnSelectedTransferChanged(null);
+    }
+
+    [RelayCommand(CanExecute = nameof(ValidateShowCommand))]
+    private void OnShow()
+    {
+        var reveal = RevealServiceFactory.get();
+        foreach (var item in SelectedTransfers)
+        {
+            if (item.Local != null)
+            {
+                foreach (var file in item.Roots)
+                {
+                    reveal.reveal(file.TransferItem.Local);
+                }
+            }
+        }
+    }
+
+    [RelayCommand(CanExecute = nameof(ValidateStopCommand))]
+    private void OnStop()
+    {
+        foreach (var item in selectedTransfers)
+        {
+            if (!item.Running)
+            {
+                continue;
+            }
+
+            item.Cancel();
+        }
+    }
+
+    [RelayCommand(CanExecute = nameof(ValidateTrashCommand))]
+    private void OnTrash()
+    {
+        foreach (var item in selectedTransfers)
+        {
+            if (!item.Running)
+            {
+                foreach (var file in item.Transfer.Roots.Items)
+                {
+                    file.Delete();
+                }
+            }
+        }
+    }
+
+    private void UpdateSelectedBandwidth()
+    {
+        if (SelectedTransfer is null)
+        {
+            return;
+        }
+
+        if (SelectedBandwidth?.Enabled == false)
+        {
+            bandwidthLookup.Remove(SelectedBandwidth.Bandwidth);
+            Bandwidth.Remove(SelectedBandwidth);
+        }
+
+        var rate = (int)SelectedTransfer.Transfer.BandwidthRate;
+        if (!bandwidthLookup.TryGetValue(rate, out var selection))
+        {
+            bandwidthLookup[rate] = selection = new(rate, false, BandwidthProvider.Format(rate));
```

Comment:
Adding a fallback element with the currently configured rate.
This element allows for changed values using default.properties, or editing user.config.